packages/waku/src/lib/handlers/dev-worker-impl.ts

```diff
@@ -26,17 +27,14 @@ const HAS_MODULE_REGISTER = typeof module.register === 'function';
 if (HAS_MODULE_REGISTER) {
   module.register('waku/node-loader', pathToFileURL('./'));
 }
-const controllerMap = new Map<number, ReadableStreamDefaultController>();
 
-const handleRender = async (mesg: MessageReq & { type: 'render' }) => {
+const handleRender = async (
+  mesg: MessageReq & { type: 'render' },
+  stream: ReadableStream,
```

Comment:
I was thinking we can simply use `rr.stream`. Any issue with it?

Reply:
you mean just use what's coming from the message?

Reply:
Yes, can we do it?

Reply:
I'll try my best!